helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/steps/MakerCheckerSteps.java

```diff
@@ -0,0 +1,135 @@
+package org.symphonyoss.symphony.bots.helpdesk.bot.it.steps;
+
+import static org.junit.Assert.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.jbehave.core.annotations.Then;
+import org.jbehave.core.annotations.When;
+import org.springframework.stereotype.Component;
+import org.symphonyoss.client.exceptions.MessagesException;
+import org.symphonyoss.client.exceptions.SymException;
+import org.symphonyoss.symphony.bots.helpdesk.bot.health.HealthCheckFailedException;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.exception.TicketRoomNotFoundException;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.helpers.MakerCheckerHelper;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.helpers.MessageHelper;
+import org.symphonyoss.symphony.bots.helpdesk.bot.it.helpers.UserHelper;
+import org.symphonyoss.symphony.clients.model.SymAttachmentInfo;
+import org.symphonyoss.symphony.clients.model.SymMessage;
+import org.symphonyoss.symphony.clients.model.SymUser;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Class responsible for managing maker checker steps.
+ * <p>
+ * Created by crepache on 05/03/18.
+ */
+@Component
+public class MakerCheckerSteps {
+
+  private static final String ATTACHMENTS_DIR = "attachment/";
+
+  private static final String APPROVE_URL = "approveUrl";
+
+  private static final String DENY_URL = "denyUrl";
+
+  private static final String STATE = "state";
+
+  private Long initialTime = 0L;
```

Comment:
Probably you should update this variable somewhere.

Reply:
Done.

Reply:
Done.